src/OrchardCore/OrchardCore.Data/OrchardCoreBuilderExtensions.cs

```diff
@@ -38,10 +38,10 @@ public static OrchardCoreBuilder AddDataAccess(this OrchardCoreBuilder builder)
                 services.AddScoped<IModularTenantEvents, AutomaticDataMigrations>();
 
                 // Adding supported databases
-                services.TryAddDataProvider(name: "Sql Server", value: "SqlConnection", hasConnectionString: true, sampleConnectionString: "The connection string to your database instance. e.g., Server=<em>localhost</em>;Database=<em>Orchard</em>;User Id=<em>username</em>;Password=<em>password</em>", hasTablePrefix: true, isDefault: false);
+                services.TryAddDataProvider(name: "Sql Server", value: "SqlConnection", hasConnectionString: true, sampleConnectionString: "Server=<em>localhost</em>;Database=<em>Orchard</em>;User Id=<em>username</em>;Password=<em>password</em>", hasTablePrefix: true, isDefault: false);
```

Comment:
The parameters are encoded, so you can't use html is it will be outputted as is.

Reply:
Nope .. it works fine because at the end it will be rendered inside a `span`, you can try it

Reply:
`<span>` has nothing to do with what I am saying. Show me a screenshot of the output in chrome. If I can see the `<em>` as a string then it means the argument is not localized, and that would be surprising.

Reply:
Is it not encoded .. I knew what you 're talking about, have a look

![ConnectionString](https://user-images.githubusercontent.com/3237266/59543438-ef182b80-8f13-11e9-8540-d1c6e0942a27.png)

Reply:
When I used the `@T[""].Value` it is encoded

Reply:
Let me check if it's a bug in our localizer ..

Reply:
I found the reason. ASP.NET works as expected and encodes the value. The de-encoding comes from the jquery code, that assigns this string using `.html()`.

Reply:
I personally would remove the tags from the strings, and use `.text()` instead to be sure there is not unintentional decoding.

Reply:
Yep both localizer works as expected .. I just finished test our localizer

> I personally would remove the tags from the strings, and use .text() instead to be sure there is not unintentional decoding.

Ok